app/views/govuk_component/govspeak_html_publication.raw.html.erb

```diff
@@ -2,7 +2,16 @@
   govspeak_locals = { content: content }
   govspeak_locals[:direction] = local_assigns.fetch(:direction) if local_assigns.include?(:direction)
   govspeak_locals[:rich_govspeak] = local_assigns.fetch(:rich_govspeak) if local_assigns.include?(:rich_govspeak)
+  sticky_footer_html = local_assigns.fetch(:sticky_footer_html) if local_assigns.include?(:sticky_footer_html)
```

Comment:
As we've added a new argument to the `govspeak_html_publication` there should be a [corresponding fixture](https://github.com/alphagov/static/blob/master/app/views/govuk_component/docs/govspeak_html_publication.yml) to appear in the component guide.